src/libraries/Common/src/Interop/Windows/Advapi32/Interop.CreateProcessWithLogon.cs

```diff
@@ -9,8 +9,8 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, ExactSpelling = true, SetLastError = true, BestFitMapping = false, EntryPoint = "CreateProcessWithLogonW")]
-        internal static extern unsafe bool CreateProcessWithLogonW(
+        [GeneratedDllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, ExactSpelling = true, SetLastError = true, EntryPoint = "CreateProcessWithLogonW")]
```

Comment:
Do we still need `ExactSpelling`?

Reply:
Yes - we're still going through the same p/invoke entry point lookup mechanism.

Reply:
Okay. So then should we add `ExactSpelling` to `LookupPrivilegeValue` below?

Reply:
For this change, I'm just keeping the same settings that exist.

But doing that would make sense - for general clean-up/best practice changes like that, I think it would be better to target main than this feature branch.